src/main/java/de/unijena/cheminf/mortar/controller/HistogramViewController.java

```diff
@@ -939,52 +931,20 @@ private StackPane createStackPaneWithContextMenuAndStructureDisplayForBar(ImageV
         tmpNodePane.setAlignment(Pos.CENTER_RIGHT);
         MenuItem tmpCopySmilesMenuItem = new MenuItem(Message.get("HistogramViewController.MenuItemSmiles.text"));
         MenuItem tmpCopyStructureMenuItem = new MenuItem(Message.get("HistogramViewController.MenuItemStructure.text"));
-        try {
-            tmpCopySmilesMenuItem.setGraphic(new ImageView(new Image("de/unijena/cheminf/mortar/images/copy_icon_16x16.png")));
-            tmpCopyStructureMenuItem.setGraphic(new ImageView(new Image("de/unijena/cheminf/mortar/images/copy_icon_16x16.png")));
-        } catch (NullPointerException | IllegalArgumentException anException) {
-            HistogramViewController.LOGGER.log(Level.WARNING, "Copy icon for context menus could not be imported.");
-        }
```

Comment:
I have added this try-catch block here because the Image constructor throws exceptions if the URL is invalid.